### PR TITLE
Cleaning some code gaps

### PR DESCRIPTION
**Pull request by Stepa4man**
### Description
Code clean up

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/PageCache/Model/Config.php [hunk 1]
```diff
@@ -199,18 +199,18 @@ protected function _getReplacements()
...
         if (!empty($accessList)) {
-            $result = [];
+            $ipsList = [];
             $ips = explode(',', $accessList);
             foreach ($ips as $ip) {
-                $result[] = sprintf($tpl, trim($ip));
+                $ipsList[] = sprintf($tpl, trim($ip));
             }
-            return implode("\n", $result);
+            return implode('\n', $ipsList);
```
Comment (lbajsarowicz) [inline app/code/Magento/PageCache/Model/Config.php]: That actually cannot work :-1: You cannot replace `"` with `'`

Comment (Stepa4man) [reply app/code/Magento/PageCache/Model/Config.php]: Thanks, fixed.

**Comment by Stepa4man** [comment]
@lbajsarowicz could you please take a look?

**Review by lbajsarowicz — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-5506](https://jira.corp.magento.com/browse/ENGCOM-5506) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**